assets/src/edit-story/components/library/panes/text/textSets/textSets.js

```diff
@@ -101,17 +130,12 @@ function TextSets() {
         />
       </CategoryWrapper>
       <TextSetContainer ref={ref} role="list" aria-labelledby={sectionId}>
-        <UnitsProvider
```

Comment:
This `UnitsProvider` is unneeded because the `TextSetElements` (which actually renders the text sets) has its own `UnitsProvider` within it.